src/scikit_build_cli/commands/init.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+import click
+
+__all__: list[str] = ["init"]
+
+
+def __dir__() -> list[str]:
+    return __all__
+
+
+@click.command()
+@click.help_option("--help", "-h")
+@click.pass_context
+def init(ctx: click.Context) -> None:  # noqa: ARG001
+    """
+    Add scikit-build to an existing project
```

Comment:
Existing CMake or Python project? Or have to have both?

Reply:
I'm not totally sure. :) I was thinking it could be like new but not overwrite existing files, and maybe adapt to some extent to what is already written.

Reply:
CMake wise it could ve tricky as everyone is having a different format. We would need a build-dir if we want to do some parsing to find the libraries that link to python-devel. Or we could `add_subdirectory` with an example path.

Python-side though should be easier to introspect, but custom `setup.py` might be harder to figure out the intent?

Or we could assume the project is either pure-python or pure-cmake for this case? That could be the simplest to support.